### PR TITLE
Add page for PECO integration

### PR DESCRIPTION
**Pull request by IceBotYT**
## Proposed change

I am adding a page that documents how to use the PECO integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65194
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3124

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by mrand** [comment]
nws_alerts documentation page is included in this pull request.  Does this new integration provide nws_alerts, or is that just left over from the previous effort?

**Comment by IceBotYT** [comment]
That’s just leftovers. Do you want me to revert it?

**Comment by mrand** [comment]
If it's unrelated, I don't think this PR can be committed until it's removed.

**Comment by IceBotYT** [comment]
Alright I’ll revert it and push. One moment

**Comment by IceBotYT** [comment]
I have reverted the NWS Alerts page

File: source/_integrations/peco.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: platinum
+ha_config_flow: true
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: peco
+---
+
+The PECO Outage Counter integration allows you to get the current count of power outages for the county, and total for the operating region too.
```
Comment (frenck) [inline source/_integrations/peco.markdown]:
```suggestion
The PECO Outage Counter integration allows you to get the current count of power outages for the county, and the total for the operating region too.
```

File: source/_integrations/peco.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ha_release: "2022.2.0"
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: platinum
+ha_config_flow: true
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: peco
+---
+
```
Comment (frenck) [inline source/_integrations/peco.markdown:13]: - Can we add something like links to the sources too?
- Maybe extend the story a little. For example, what this integration can do/is useful for.
- A comment for the countries this applies to might be helpful. I'm pretty sure the counties in my country, aren't covered 😉

File: source/_integrations/peco.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+The PECO Outage Counter integration allows you to get the current count of power outages for the county, and total for the operating region too.
+
+{% include integrations/config_flow.md %}
+
+<div class='note'>
+
+When you are configuring the skill, the county named `TOTAL` creates a sensor which gets the total of all of the counties in the region.
+
+</div>
```
Comment (frenck) [inline source/_integrations/peco.markdown]: - Does this have to be a big info box?
- `TOTAL` isn't code and should not be code fenced.
- "TOTAL" is capitalized, which seems kinda shouty/unfriendly. Can we adjust that?

File: source/_integrations/peco.markdown [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: "PECO Outage Count"
+description: "Get how many outages are in your county"
+ha_release: "2022.2.0"
```
Comment (frenck) [inline source/_integrations/peco.markdown]:
```suggestion
ha_release: 2022.4
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/peco.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: peco
+---
+
+PECO is a public utility company that provides electricity to the Philadelphia metro.
+
+To learn more about PECO, visit [**their about page**](https://www.peco.com/AboutUs/Pages/Default.aspx).
+
+<div class='note warning'>
```
Comment (frenck) [inline source/_integrations/peco.markdown]: Just a note will do 😉 

```suggestion
<div class='note'>
```

File: source/_integrations/peco.markdown [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+The PECO Outage Counter integration allows you to get the current count of power outages for any county, and the total for the operating region too.
+
+This integration will create 4 entities:
+
+- `sensor.<county>_outage_count`
+- `sensor.<county>_customers_out`
+- `sensor.<county>_customers_served`
+- `sensor.<county>_percent_customers_out`
```
Comment (frenck) [inline source/_integrations/peco.markdown]: Entity IDs are not guaranteed and, above all, can be changed. Instead, just describe the sensors instead.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @IceBotYT 👍

**Pull request merged**